### ISSUE TITLE
Erythema Marginatum

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#10579]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**
PMID: 32491767 Erythema marginatum is reactive inflammatory erythema seen most commonly in association with acute rheumatic fever. Although a rare cutaneous manifestation, it is of utmost diagnostic value for acute rheumatic fever as well as other rare disorders. It can be described as an evanescent, blanchable, nonpruritic macular rash seen generally on the trunk and extremities. It can also be seen in hereditary angioedema and psittacosis.

PMID: 29489144  The attacks last 2 to 5 days, usually slowly increasing and then resolving even without therapy.  Triggers, such as trauma, infections, stress, or procedures, are identified in 40% of attacks. Prodromal symptoms, including a rash, referred to as **erythema marginatum**, occur before the majority of attacks, but the sensitivity and especially specificity are questionable. Other triggers include ACE inhibitors and estrogens.

PMID: 27503784 One-third of patients may develop an erythematous, nonpruritic rash, **erythema marginatum**, which might precede or accompany angioedema, although it can also occur independently

**Parent term (use hpo.jax.org/app)**
Erythema HP:0010783

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:528623 Hereditary angioedema with C1Inh deficiency

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6001012
- label: Erythema marginatum
- parents: [Erythema (HP:0010783)](https://hpo.jax.org/app/browse/term/HP:0010783)
- pmid: PMID:32491767
- def: An erythematous (pink-red, depending on the skin color of the affected individual) macule that spreads peripherally and coalesces to form polycyclic or serpentine patches or plaques. Lesions occur in crops over days to weeks and are evanescent. The rate of migration is 2 to 12 mm over a period of a few hours. Erythema marginatum tends to affect the trunk more than the extremities and to not be accompanied by pruritus.
- comment: 
- orcid: orcid:0009-0005-6714-5727
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10579
- synonyms:
